wallet/lib/pages/home/home.dart

```diff
@@ -118,7 +115,7 @@ class HomeScreenState extends State<HomeScreen> with SingleTickerProviderStateMi
 
   PreferredSize buildAppBar(BuildContext context, HomeProvider provider) {
     return PreferredSize(
-      preferredSize: Size.fromHeight(isTablet ? 0.4.sh : 0.2.sh + 115.h),
+      preferredSize: Size.fromHeight(isTablet ? 0.3.sh : 0.2.sh + 40.h),
```

Comment:
Did you checked on Tablet?

Reply:
yes I have checked it... screenshots are attached as well.